web/themes/custom/votegov/templates/navigation/menu--footer.html.twig

```diff
@@ -23,7 +23,7 @@
 {% import _self as menus %}
 
 {# footer links #}
-<nav class="usa-identifier__section usa-identifier__section--required-links" data-test="footer">
+<nav class="usa-identifier__section usa-identifier__section--required-links" aria-label="{{ 'Secondary navigation' | t }}" data-test="footer">
```

Comment:
This was the updated that was requested to move to the nav and off of the footer.. i added here since this was the nav that wrapped the links.

Reply:
@SamiraMSadat this is a one off fix that Ray asked for.. i made and update to the Primary Nav so just did it here as well this was just the placement of the label

Reply:
Okay, then this will have to be a separate bug ticket.

Reply:
I created this ticket!